pkg/nsx/services/vpc/vpc.go

```diff
@@ -233,204 +238,6 @@ func (s *VPCService) DeleteVPC(path string) error {
 	return nil
 }
 
-func (s *VPCService) addClusterTag(query string) string {
```

Comment:
Why removed all these new changes?

Reply:
it's due to wrong operation while switch branch. fixed.